rosidl_generator_cpp/resource/msg__traits.hpp.em

```diff
@@ -84,14 +87,14 @@ inline void to_yaml(
 @[    if isinstance(member.type, BasicType)]@
     out << "@(member.name): ";
 @[      if member.type.typename in ('octet', 'char', 'wchar')]@
-    character_value_to_yaml(msg.@(member.name), out);
+    rosidl_generator_traits::character_value_to_yaml(msg.@(member.name), out);
```

Comment:
I'm not sure if we should keep `character_value_to_yaml()` in `rosidl_generator_traits` or if we should move it to another place

Reply:
So, they have to be in a common place. I don't mind where. I figured keeping them where they are was the path of least resistance, but I'm open to ideas.

Reply:
sounds good.
rosidl_generator_traits doesn't sound like the namespace for this helper functions, but it doesn't really matter.